bylaws.md

```diff
@@ -88,7 +88,7 @@ c) The corporation adopts and implements some means of verifying (1) that all pe
 
 ### SECTION 8. REGULAR AND ANNUAL MEETINGS ###
 
-Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday, in which event the regular meeting shall be held at the same hour and place on the next business day.
+Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM Central Time, unless such day falls on a legal holiday or a significant number of directors are unable to attend, in which event the regular meeting shall be held at the same hour and place on the next business day or another generally agreed upon day.
```

Comment:
Historically we have changed our meeting cadence, and it's pure luck that we landed back on Wednesday @ 2 pm again. Maybe we should note that when the board changes, we poll the board for a monthly time that works for the group. We also reserve the right to not meet over the holidays or when a majority fo the board agrees we don't need to.